app/uk/gov/hmrc/customsservicestatusfrontend/controllers/DashboardController.scala

```diff
@@ -52,11 +54,20 @@ class DashboardController @Inject() (
       val stateChangedAt = statuses.services.find(_.state.contains(UNAVAILABLE)).flatMap(_.stateChangedAt).getOrElse(Instant.now())
 
       val today = LocalDate.now(ZoneId.systemDefault())
```

Comment:
This needs to be set to Europe/London, otherwise it won't work correctly during British Summer Time.